.github/workflows/build.yml

```diff
@@ -85,7 +85,9 @@ jobs:
     - id: ctest
       name: ctest
       working-directory: ${{env.BuildDir}}
-      run: ctest -C ${BuildType} -V -T Test --no-compress-output -R python-test ${{matrix.ctest_extra}}
+      run: |
+        export LD_LIBRARY_PATH=${{env.InstallPrefix}}/lib64
```

Comment:
OK, so here you are trying to solve a third ticket (#7 ) in a single PR. Please don't, that's just too much going on at the same time for me ;P

Reply:
Sorry for that, Shall I remove the this changes and push it again? I'll created separate PR for #7

Reply:
Yes please.

Reply:
It might be best if we finish the discussion on #7 about the strategy how to fix this. As I said in a comment there, messing with LD_LIBRARY_PATH will not get you there, because `cproton` is not even being built!

Reply:
okay